control-plane/agents/src/bin/core/node/specs.rs

```diff
@@ -183,4 +185,45 @@ impl ResourceSpecsLocked {
         registry.store_obj(&drained_node_spec).await?;
         Ok(drained_node_spec)
     }
+
+    pub(crate) async fn add_node_draining_volume(
```

Comment:
Can we add some doc comments as the methods are `pub(crate)`. Below as well.

Reply:
fixed